app/lib/next_of_kin_registration_validator.rb

```diff
@@ -0,0 +1,7 @@
+module NextOfKinRegistrationValidator
+  def next_of_kin_needed_to_register?
+    # TODO: Put survey config in in yaml file
```

Comment:
This depends on https://github.com/Australian-Genomics/CTRL/pull/56 being merged.